client/tests/integration/roles.rs

```diff
@@ -1,102 +1,13 @@
 #![allow(clippy::restriction)]
 
-use std::{str::FromStr as _, time::Duration};
+use std::str::FromStr as _;
 
-use eyre::{eyre, Result};
-use iroha_client::client::{self, Client};
+use eyre::Result;
+use iroha_client::client::{self};
 use iroha_core::prelude::*;
 use iroha_data_model::prelude::*;
-use iroha_permissions_validators::public_blockchain::{
-    key_value::{CanRemoveKeyValueInUserMetadata, CanSetKeyValueInUserMetadata},
-    transfer,
-};
 use test_network::*;
 
-#[ignore = "ignore, more in #2851"]
-#[test]
-fn add_role_to_limit_transfer_count() -> Result<()> {
```

Comment:
this is intentionally removed?

Reply:
Yes, it doesn't make sense anymore. If user want to create such a limitation when they should write a specific validator for that